### DISCUSSION TITLE
`ltex-ls` formula unnecessarily depends on openjdk?

### DISCUSSION THREAD

**Discussion by fhfuih** [Homebrew/discussions#3225] [Everyday usage] [answered]
I saw the [ltex-ls formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/ltex-ls.rb) on homebrew requires openjdk. But the [official document](https://valentjn.github.io/ltex/ltex-ls/installation.html) says

> It’s recommended that you choose the archive corresponding to your platform (these archives are standalone, no Java installation necessary).
> If you choose the platform-independent file ltex-ls-VERSION.tar.gz, then you need Java 11 or later on your computer.

I wonder if homebrew's CI builds bottles the same way as the developer, as the bottles are indeed platform-dependent. If it does, then will the JDK dependency be removed in the future?

**Comment by SMillerDev** [comment, marked as answer]
Generally we try to stick to the upstream build instructions. But the platform archives there just package their own JDK so they're not really "no Java installation necessary", they're more "no _standalone_ Java installation necessary"
